components/lists/lists.css

```diff
@@ -0,0 +1,14 @@
+.list-component{
+
+    padding: 1rem 2rem;
+    background-color: var(--white-color);
+}
+
+.indent-x{
+    padding: 0 1rem ;
+}
+
```

Comment:
use utility classes which you have defined

Reply:
done